.gitignore

```diff
@@ -1,72 +1,3 @@
-# Xcode
```

Comment:
basically none of this was relevant at all, this is likely what got generated for new Swift projects back around the time of Swift 4, so I cleaned it up to only include actually relevant things